### ISSUE TITLE
CIS-DI-0009 check no longer works when using docker build kit

### ISSUE THREAD

**Issue by philnichol** [goodwithtech/dockle#134]
First off, thanks for creating and maintaining this awesome project! We've been using it recently and it's been great 🙂  

**Description**

Dockle doesn't flag CIS-DI-0009 when the image being scanned was built with docker buildkit.

**What did you expect to happen?**
When I run dockle <IMAGE> with an image that uses `ADD` instead of `COPY`, dockle outputs `FATAL	- CIS-DI-0009: Use COPY instead of ADD in Dockerfile`

**What happened instead?**
When I run dockle <IMAGE> with an image that uses `ADD` instead of `COPY`, dockle doesn't output any FATAL or WARNING messages

**Output of run with `-debug`:**

```
$ dockle -debug test:1
2021-08-09T15:36:26.885+0100	DEBUG	There is no .dockleignore file
2021-08-09T15:36:26.902+0100	DEBUG	Fetch latest version from github
2021-08-09T15:36:27.652+0100	INFO	Failed to check latest version. not found version patterns
2021-08-09T15:36:27.652+0100	DEBUG	Start assessments...
2021-08-09T15:36:40.999+0100	DEBUG	Start scan : password files
2021-08-09T15:36:40.999+0100	DEBUG	Start scan : /etc/passwd
2021-08-09T15:36:40.999+0100	DEBUG	Start scan : /etc/group
2021-08-09T15:36:40.999+0100	DEBUG	Start scan : /etc/hosts
2021-08-09T15:36:40.999+0100	DEBUG	Start scan : credential files
2021-08-09T15:36:40.999+0100	DEBUG	Scan start : config file
2021-08-09T15:36:40.999+0100	DEBUG	Scan start : DOCKER_CONTENT_TRUST
2021-08-09T15:36:40.999+0100	DEBUG	Start scan : cache files
2021-08-09T15:36:40.999+0100	DEBUG	End assessments...
WARN	- CIS-DI-0001: Create a user for the container
	* Last user should not be root
INFO	- CIS-DI-0005: Enable Content trust for Docker
	* export DOCKER_CONTENT_TRUST=1 before docker pull/build
INFO	- CIS-DI-0006: Add HEALTHCHECK instruction to the container image
	* not found HEALTHCHECK statement
INFO	- DKL-LI-0003: Only put necessary files
	* unnecessary file : Dockerfile
```

**Output of `dockle -v`:**

```
$ dockle -v
dockle version 0.3.15
```

**Additional details (base image name, container registry info...):**
If I build the image with `DOCKER_BUILDKIT=0`, then dockle is able to check the history correctly. `DOCKER_BUILDKIT=1` is set to 1 as a default. 

**Contents of Dockerfile**
```
$ cat Dockerfile
FROM alpine

ADD . .
``` 

**Working example when unsetting the docker buildkit var**
```
$ DOCKER_BUILDKIT=0 docker build -t test:1 .
Sending build context to Docker daemon    747MB
Step 1/2 : FROM alpine
latest: Pulling from library/alpine
29291e31a76a: Pull complete
Digest: sha256:eb3e4e175ba6d212ba1d6e04fc0782916c08e1c9d7b45892e9796141b1d379ae
Status: Downloaded newer image for alpine:latest
 ---> 021b3423115f
Step 2/2 : ADD . .
 ---> 6ef15e4b5a88
Successfully built 6ef15e4b5a88
Successfully tagged test:1

$ dockle -debug test:1
2021-08-09T15:40:23.800+0100	DEBUG	There is no .dockleignore file
2021-08-09T15:40:23.807+0100	DEBUG	Fetch latest version from github
2021-08-09T15:40:24.498+0100	INFO	Failed to check latest version. not found version patterns
2021-08-09T15:40:24.499+0100	DEBUG	Start assessments...
2021-08-09T15:40:37.972+0100	DEBUG	Start scan : password files
2021-08-09T15:40:37.972+0100	DEBUG	Start scan : /etc/passwd
2021-08-09T15:40:37.972+0100	DEBUG	Start scan : /etc/group
2021-08-09T15:40:37.972+0100	DEBUG	Start scan : /etc/hosts
2021-08-09T15:40:37.972+0100	DEBUG	Start scan : credential files
2021-08-09T15:40:37.972+0100	DEBUG	Scan start : config file
2021-08-09T15:40:37.972+0100	DEBUG	Scan start : DOCKER_CONTENT_TRUST
2021-08-09T15:40:37.972+0100	DEBUG	Start scan : cache files
2021-08-09T15:40:37.972+0100	DEBUG	End assessments...
FATAL	- CIS-DI-0009: Use COPY instead of ADD in Dockerfile
	* Use COPY : /bin/sh -c #(nop) ADD dir:ffc8e233e663b1758b125145eea2e3e78be1950c842075d19e02507bd7e1e369 in .
WARN	- CIS-DI-0001: Create a user for the container
	* Last user should not be root
INFO	- CIS-DI-0005: Enable Content trust for Docker
	* export DOCKER_CONTENT_TRUST=1 before docker pull/build
INFO	- CIS-DI-0006: Add HEALTHCHECK instruction to the container image
	* not found HEALTHCHECK statement
INFO	- DKL-LI-0003: Only put necessary files
	* unnecessary file : Dockerfile
```

**Comment by tomoyamachi** [comment]
Not released yet.

**Comment by tomoyamachi** [comment]
@philnichol 
Thank you for the report!
I fixed it on v0.3.16.
